src/Classes/PolygonMask.ts

```diff
@@ -0,0 +1,227 @@
+"use strict";
+
+import {Container} from "./Container";
+import {ICoordinates} from "../Interfaces/ICoordinates";
+import {PolygonMaskType} from "../Enums/PolygonMaskType";
+import {Particle} from "./Particle";
+
+// @ts-ignore
+declare function require(name:string);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'require' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5042590)